modules/chapter1/pages/section2.adoc

```diff
@@ -2,34 +2,60 @@
 :navtitle: Architecture
 
 
-Red{nbsp}Hat OpenShift Data Science (RHODS) is a collection of tools that run on top of OpenShift.
+Red{nbsp}Hat OpenShift Data Science (RHODS) is a unified platform for data scientists, developers, and IT ops.
+The platform is designed to manage the complete lifecycle of AI/ML projects in the hybrid cloud.
 
-*TODO: improve this diagram*
+As the following diagram shows, RHODS is deployed as an operator, and includes several components to handle AI/ML development/training, automation pipelines, and model serving:
 
 image::architecture.svg[width=800px]
```

Comment:
This diagram is excellent ! but size can be bigger. I have a hard time making out the small text. Maybe sak Sajith to expand and clean up?

Reply:
Request created https://redhat.service-now.com/help?id=rh_ticket&table=x_redha_red_hat_tr_x_red_hat_training&sys_id=5bfb746edbf13d54c4db45e813961975